### PR TITLE
Support the updated legal-docs repo layout

### PR DESCRIPTION
**Pull request by pmac**
See https://github.com/mozilla/legal-docs/issues/1501

### CODE REVIEW

**Comment by pmac** [comment]
Draft for now as the restructuring timeline in the linked issue hasn't been decided yet. Though this change works with both structures, so we could merge this at any time.

**Comment by pmac** [comment]
@peiying2 has indicated that we'd like to test this new layout in bedrock. When we merge this we can configure dev to use the `restructure` branch in the legal-docs repo so we can ensure the entire system is working as intended.

File: bedrock/legal_docs/models.py [hunk 1]
```diff
@@ -32,16 +34,25 @@ def process_md_file(file_path):
 
 
 def get_data_from_file_path(file_path):
-    locale = file_path.stem
+    print(file_path)
```
Comment (glogiotatidis) [inline bedrock/legal_docs/models.py:37]: nit: remove this line

**Review by glogiotatidis — APPROVED** [review]
This slipped through the cracks, apologies for not reviewing earlier. Looks good!

File: bedrock/legal_docs/models.py (hunk 1 above)
Comment (pmac) [reply bedrock/legal_docs/models.py:37]: Doh! Good catch.

**Comment by pmac** [comment]
Okay. Legal-docs is ready to go. Merging.

**Pull request merged**